ras_party/controllers/respondent_controller.py

```diff
@@ -46,7 +46,7 @@ def get_respondent_by_ids(ids, session):
     :rtype: Respondent
     """
     respondents = query_respondent_by_party_uuids(ids, session)
-    return [respondent.to_respondent_with_associations_dict() for respondent in respondents]
+    return [respondent.to_respondent_dict() for respondent in respondents]
```

Comment:
I've checked all service and I'm confident no-where uses the associations element

Reply:
What was the resolution following this comment?

Reply:
It's been fixed, there is a card in rops